frappe/desk/form/meta.py

```diff
@@ -36,7 +36,7 @@
 
 def get_meta(doctype, cached=True) -> "FormMeta":
 	# don't cache for developer mode as js files, templates may be edited
-	cached = cached and not frappe.conf.developer_mode
+	cached = cached and not frappe._dev_server
```

Comment:
This change is not equivalent - one relates to having DEV_SERVER envvar set in the context of a given process and the other relates only  a "site" level

Reply:
Why would you specify `conf.developer_mode` only on specific sites? I don't get the use-case.